autotest/client/go/goclienttest/TestCache/const.go

```diff
@@ -38,4 +38,5 @@ const (
 	messagevalue     = "Your contact page."
 	frmessagevalue   = "Votre page de contact."
 	dlcnmessagevalue = "您的联系人页面。"
+	bat_path         = "E:/234/goclienttest/go_bat/"
```

Comment:
It seems the bat_path is hard coded, please double confirm

Reply:
the bat path not use in smoke test and i will change it to other path